examples/demo-react-native-jest/e2e/config.json

```diff
@@ -1,4 +1,6 @@
 {
   "setupFilesAfterEnv": ["./init.js"],
-  "testEnvironment": "node"
+  "testEnvironment": "node",
+  "reporters": ["detox/runners/jest/DetoxJestReporter"],
```

Comment:
What if we alias or just rename it to `detox/runners/jest/reporter`? It will look a bit sleeker.